libs/SalesforceHybridSDK/SalesforceHybridSDK.xcodeproj/project.pbxproj

```diff
@@ -1506,6 +1506,7 @@
 			baseConfigurationReference = CE8F13091AD4A87200233F2F /* SalesforceHybridSDK-Test.xcconfig */;
 			buildSettings = {
 				BUNDLE_LOADER = "$(BUILT_PRODUCTS_DIR)/SalesforceHybridSDKTestApp.app/SalesforceHybridSDKTestApp";
+				CLANG_ALLOW_NON_MODULAR_INCLUDES_IN_FRAMEWORK_MODULES = YES;
```

Comment:
We've been trying pretty hard to get rid of these.  What's triggering the requirement this time?